cores/esp32/esp32-hal-uart.c

```diff
@@ -401,7 +401,12 @@ uint32_t uartGetBaudRate(uart_t* uart)
     if(uart == NULL) {
         return 0;
     }
+
     uint32_t clk_div = (uart->dev->clk_div.div_int << 4) | (uart->dev->clk_div.div_frag & 0x0F);
+    if(!clk_div) {
```

Comment:
in what case is `clk_div` == 0?

Reply:
I do not know, but I got a division by zero error and stack trace pointed to uartGetBaudRate(). It was a 'random' error, not reproducible...

Reply:
Interesting... ok :)